### ISSUE TITLE
MIOU effect on the dataset

### ISSUE THREAD

**Issue by Chihanxx** [VainF/DeepLabV3Plus-Pytorch#101]
Hello Author.
I use resnet101 as the backbone of the network and use it to train on cityscapes dataset, the miu metric I get is only 73.4, which is much worse than the 76.2 you mentioned. Is there something wrong with me?
The following is my training code.

python main.py --model deeplabv3plus_resnet101 --dataset cityscapes --enable_vis --vis_port 28333 --gpu_id 0,1  --lr 0.1  --crop_size 768 --batch_size 16 --output_stride 16 --data_root ./datasets/data/cityscapes

**Comment by DC-SegNet** [comment]
l have same issue,l use mobilenetv2 as the backbone of the network and use it to train on voc12,l used the pretrained model,but the miou l get is only 68.6,witch is much worse than 0.711 you mentioned. @VainF

**Comment by 18109359681** [comment]
> 你好作者。我使用 resnet101 作为网络的骨干，并用它来训练城市景观数据集，我得到的 miu 指标只有 73.4，比你提到的 76.2 差得多。我有什么问题吗？以下是我的训练代码。
> 
> python main.py --model deeplabv3plus_resnet101 --dataset cityscapes --enable_vis --vis_port 28333 --gpu_id 0,1 --lr 0.1 --crop_size 768 --batch_size 16 --output_stride 16 --data_root ./datasets/data/cityscapes

你好，问题解决了吗
